workloads/network-perf/smoke_test.sh

```diff
@@ -14,3 +14,27 @@ delete_benchmark
 done
 print_uuid
 generate_csv
+
+if [[ ${ENABLE_SNAPPY_BACKUP} == "true" ]] ; then
+ echo -e "snappy server as backup enabled"
+ csv_list=`find . -name "*.csv"` 
+ mkdir files_list
+ cp $csv_list ./files_list
+ tar -zcvf snappy_files.tar.gz ./files_list
+ 
+ export workload=network_perf_smoke_test
+ export platform=$(oc get infrastructure cluster -o jsonpath='{.status.platformStatus.type}')
```

Comment:
Maybe make this part of a common function? This way we can reuse it across all the workloads. Thoughts?

Reply:
I have created a common functions file for every workload in this pr - https://github.com/cloud-bulldozer/e2e-benchmarking/pull/145 .. can add them there once merged

Reply:
@chaitanyaenr created a common.sh in snappy-move-results directory and we call that in all the workloads which need snappy integration